package.json

```diff
@@ -6,16 +6,19 @@
     "yarn": ">=1.16.0"
   },
   "scripts": {
+    "build-prod": "cross-env NODE_ENV=production webpack --display-error-details -p --config webpack.config.js",
+    "build:server:prod": "cross-env NODE_ENV=production webpack --display-error-details --config webpack.config.server.js",
+    "server:migrate": "cross-env MIGRATE_ONLY=true node --experimental-worker dist/server.js",
     "build": "webpack --display-error-details -p --config webpack.config.js",
-    "build-prod": "cross-env-shell NODE_ENV=production BUILD_REPORT=true webpack --display-error-details -p --config webpack.config.js",
+    "build-prod-report": "cross-env NODE_ENV=production BUILD_REPORT=true webpack --display-error-details -p --config webpack.config.js",
     "build:watch": "webpack --display-error-details --config webpack.config.js --watch --experimental-worker",
     "clean": "rm -rf dist/",
-    "start": "node server/server.js",
+    "start": "node --experimental-worker dist/server.js",
     "postinstall": "webpack --display-error-details --config webpack.config.js",
-    "watch": "npm run build:server:once && npm-run-all --parallel build:watch nodemon:prod watch:server",
+    "watch": "npm run build:server:dev && npm-run-all --parallel build:watch nodemon:prod watch:server",
     "create-migration": "./server/db/migration/create-migration.sh",
-    "build:server:once": "cross-env NODE_ENV=development webpack --display-error-details --config webpack.config.server.js",
-    "dev:server": "npm run build:server:once && npm-run-all --parallel nodemon:prod watch:server",
+    "build:server:dev": "cross-env NODE_ENV=development webpack --display-error-details --config webpack.config.server.js",
+    "dev:server": "npm run build:server:dev && npm-run-all --parallel nodemon:prod watch:server",
     "watch:server": "cross-env NODE_ENV=development webpack --display-error-details --config webpack.config.server.js --watch",
     "nodemon:prod": "cross-env NODE_ENV=development node-dev --no-notify --experimental-worker --inspect -- dist/server.js",
     "test": "mocha-webpack --timeout 30000 --display-error-details --webpack-config webpack.config.test.js \"test/*/*.js\""
```

Comment:
can we run test with its own postgres docker container ? this way we won't need ```server:migrate```?

Reply:
we still need server:migrate